lib/stat_tracker.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_literal_string: true
+
```

Comment:
We need to remove comments, Probably can be done at a later pull.

Reply:
good catch!